shapeshift/web
# feat: polygon support

## Description

This pull request adds support for Polygon, a new blockchain, to the ShapeShift platform. With this update, users can now track their Polygon balances and swap their assets to and from Polygon from other supported blockchains.

This new feature provides users with greater flexibility and choice when managing their crypto assets, especially as Polygon's popularity continues to grow.

## Pull Request Type


## Risk

Because CAIP and asset metadata was regenerated, this might impact other chains if something was been messed up.

## Testing

To test this feature, follow these steps:

- Connect a wallet that supports Polygon
- Verify that Polygon is listed
- Swap some tokens between other chains and Polygon
- Confirm that trades work correctly
- Confirm that asset tracking works correctly, all Polygon ERC20 tokens should be supported


## Screenshots (if applicable)
![Zrzut ekranu 2023-04-19 o 16 42 21](https://user-images.githubusercontent.com/5132069/233111766-eec87c29-9f7d-4699-9f50-59d391093fc3.png)
![Zrzut ekranu 2023-04-19 o 16 42 37](https://user-images.githubusercontent.com/5132069/233111807-0afcf5c2-b683-4b53-b9c9-94f4fa77165d.png)
![Zrzut ekranu 2023-04-19 o 16 42 04](https://user-images.githubusercontent.com/5132069/233111834-50a4fc5d-613c-4a52-b658-e88b438f2f8b.png)
